packages/builder/src/pages/builder/admin/index.svelte

```diff
@@ -83,9 +94,13 @@
           validate={() => {
             let fieldError = {}
 
-            fieldError["password"] = !formData.password
-              ? "Please enter a password"
-              : undefined
+            if (!formData.password) {
+              fieldError["password"] = "Please enter a password"
+            } else if (formData.password.length < 12) {
```

Comment:
can we remove this and just depend on the backend validation of this - so that dev environment respects it?

Reply:
Updated admin store to expose `passwordMinLength`. Example:

![example](https://github.com/user-attachments/assets/d5a39576-4db4-4963-9fdc-f853292fe93e)

Defaults to 12 if the variable is missing.